### ISSUE TITLE
Add Support for Top Level ARS IDs to GUI

### ISSUE THREAD

**Issue by MarkDWilliams** [RTXteam/RTX#1231]
Thanks very much for adding ARS ID support to the ARAX GUI.  I'd like to request that this functionality be extended to "top-level" IDs for queries.  In the ARS, when a query is received, it goes first to the ARS Default Agent, which then sends it out to the other registered services.  This message from the ARS Default Agent is technically similar to those provided by individual responses, but requires the additional `trace=y` parameter at the end of the URL.  The default agent's message has no content of its own other than a series of children, each of which is one of the individual responses that are already supported by the GUI.  This can be seen here
https://ars.transltr.io/ars/api/messages/828551fb-c4dc-4851-8705-762520a5c428?trace=y
```
{
message: "828551fb-c4dc-4851-8705-762520a5c428",
status: "Done",
actor: {
pk: 9,
channel: "general",
agent: "ars-default-agent",
path: ""
},
children: [
{
message: "f3130918-7321-4434-9c8b-1e56f8d90a87",
status: "Done",
actor: {
pk: 1,
channel: "general",
agent: "ara-aragorn",
path: "runquery"
},
children: [ ]
},
{
message: "62a34151-c5fc-4b0b-be65-f29b5e2eba2e",
status: "Done",
actor: {
pk: 2,
channel: "general",
agent: "ara-arax",
path: "runquery"
},
children: [ ]
},
{
message: "78f01898-c988-44ab-b092-1fd11d93fa11",
status: "Running",
actor: {
pk: 3,
channel: "general",
agent: "ara-bte",
path: "runquery"
},
children: [ ]
}
]
}
```

If possible, it would be great to extend the GUI support for individual IDs to also allow a user to input one of these top-level IDs and then view each/all of the contained result sets.  I'm happy to provide more detail or clarification where helpful with any of this.  Thanks!

**Comment by edeutsch** [comment]
Hi Mark, sure, Luis cobbled a little something together, Try going to the "id" tab at the usual place:
https://arax.ncats.io/

and then paste in a collection GUID like you list above: 828551fb-c4dc-4851-8705-762520a5c428
and click LOAD.
You should see the list of resources and the GUIDs for each individual response. You can click on those and then
click on Results on the left if there are any. Sometimes takes a few second to load.
TRAPI validation errors are displayed at the bottom if there are any.

Let us know how it works out and report and bugs you find.

**Comment by isbluis** [comment]
Addressed in commit 9e65ee221d0b75623914b5b3feb1528722924dff

**Comment by MarkDWilliams** [comment]
Looks to be working fine.  Thanks!
